actionRuntimes/bashAction/actionproxy.py

```diff
@@ -0,0 +1,298 @@
+"""Executable Python script for a proxy service to bashAction.
```

Comment:
do you need actionproxy vs just overriding the `ActionRunner`? See https://github.com/apache/incubator-openwhisk-runtime-swift/blob/master/core/swift3.1.1Action/swift3runner.py#L113 for example.